.github/workflows/ci.yaml

```diff
@@ -215,6 +214,9 @@ jobs:
           disable-animations: false
           script: echo "Generated AVD snapshot for caching."
       
+      - run: yarn install
+      - run: nohup yarn react-native start > "out.yarn_react_native_start" &
```

Comment:
oh we missed react-native start! default output for nohup in `nohup.out` so should this be `yarn_react_native_start.out`?